aopy/data/db.py

```diff
@@ -114,6 +115,34 @@ def lookup_bmi_sessions(bmi_task_name='bmi control', **kwargs):
     '''
     return lookup_sessions(task_name=bmi_task_name, **kwargs)
 
+def lookup_decoder_parent(task_name='nothing', task_desc='decoder parent', **kwargs):
+    '''
+    Lookup by project and session
+    '''
+    return lookup_sessions(task_name=task_name, task_desc=task_desc, **kwargs)
+
+def lookup_decoders(id=None, parent_id=None, **kwargs):
+    '''
+    Returns list of decoders with the given filter parameters
+
+    Args:
+        id (int or list, optional): Lookup decoders with the given ids, if provided.
+        parent_id (int, optional): Lookup decoders with the given parent ids, if provided.
+        kwargs (dict, optional): optional keyword arguments to pass to database lookup function.
+
+    Returns:
+        list: list of Decoder records matching the query
+    '''
+    if id is not None and isinstance(id, list):
+        kwargs['id__in'] = id
+    elif id is not None:
+        kwargs['id'] = id
+    if parent_id is not None:
+        kwargs['entry_id'] = parent_id
```

Comment:
Do you need to do the same condition if parent_id is or is not a list?

Reply:
i don't think it makes as much sense to lookup decoders from multiple parents..?